privateSBOMExchange/Evaluation/test_evaluations.py

```diff
@@ -4,18 +4,17 @@
 import sys
```

Comment:
This eval takes a while, would it be worthwhile adding tqdm to it so we know how far ahead it is?